src/components/button.tsx

```diff
@@ -107,6 +120,7 @@ export default function ButtonComponent( {
 				baseStyles,
 				variant === 'primary' && primaryStyles,
 				variant === 'secondary' && secondaryStyles,
+				variant === 'tertiary' && tertiaryStyles,
```

Comment:
Leverage `tertiary` for InlineCLI ActionButtons. (`tertiary` is currently unused as a `<Button />` variant in `trunk`.)

Reply:
I noted these new tertiary styles modified the existing usage of tertiary in the add site modal. We might consider consolidating button styles (i.e., reduce the number of button styles used) or rename this new style. WDYT? 

https://github.com/Automattic/studio/blob/216154a91b4f4e49d6d2c178b72481a96e6a3668/src/components/add-site.tsx#L140

Reply:
@dcalhoun Good catch! I think we should use a different name for the buttons in code blocks as they're a bit of a special use case. I don't anticipate on re-using those dark styles anywhere else in the app at the moment.

Reply:
On second thought, let's update these buttons to use the same styling as the `Add site` button in the sidebar (outlined, not filled). That helps bring more consistency to the button styles in the app. ([Figma](https://www.figma.com/design/RToz6tIuQ7nlZrikBte4GU/Studio?node-id=2988-45230&t=PVkuQrwOqgk9cbDO-4))

![image](https://github.com/Automattic/studio/assets/417538/9fa35621-a52e-453e-82cf-661f8d90f123)

Reply:
For posterity, I drafted https://github.com/Automattic/studio/pull/243 addressing the button styling.